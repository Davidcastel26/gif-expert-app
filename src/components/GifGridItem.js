import React from 'react'
import PropTypes from 'prop-types'

function GifGridItem({ title, url}) {

    // console.log({id, title,url});

    return (
        <div>
            {/* {img.title} */}
            <img src={url} alt={title} />
            <p> {title} </p>
        </div>
    )
}

GifGridItem.propTypes = {

}

export default GifGridItem;

