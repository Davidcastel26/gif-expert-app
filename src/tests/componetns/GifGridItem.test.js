import React from 'react'
import { shallow } from 'enzyme'
import GifGridItem from '../../components/GifGridItem'


describe('Test about Gif Grid Item component', ()=>{

    const title = 'a title';
    const url = 'https://localhost/algo.jpg' 

    test('should show the component in the right way', () => {

        const wrapper = shallow( <GifGridItem title={title} url={url}/> )
        expect( wrapper ).toMatchSnapshot();

    })


})