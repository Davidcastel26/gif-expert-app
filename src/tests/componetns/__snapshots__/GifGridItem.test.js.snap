// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test about Gif Grid Item component should show the component in the right way 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="a title"
    src="https://localhost/algo.jpg"
  />
  <p>
     
    a title
     
  </p>
</div>
`;
